compose-builder/.env

```diff
@@ -49,6 +49,7 @@ KONG_VERSION=2.8
 KUIPER_VERSION=1.7-alpine
 MOSQUITTO_VERSION=2.0
 NATS_VERSION=2.9-alpine
+NGINX_VERSION=1.23-alpine-slim
```

Comment:
Need to remove KONG_VERSION and POSTGRES_VERSION

Reply:
Fixed in https://github.com/edgexfoundry/edgex-compose/pull/345/commits/252fc9b8452f2bec2d038c013fc44d351338f6cc